platform/solr/solr-factory/src/main/java/org/codice/solr/factory/impl/HttpSolrClientFactory.java

```diff
@@ -309,6 +290,82 @@ private static void createSolrCore(
     }
   }
 
+  private HttpClientBuilder createHttpBuilder() {
+
+    HttpClientBuilder httpClientBuilder =
+        HttpClients.custom()
+            .setDefaultCookieStore(new BasicCookieStore())
+            .setMaxConnTotal(128)
+            .setMaxConnPerRoute(32);
+
+    if (useTls()) {
+      httpClientBuilder.setSSLSocketFactory(
+          new SSLConnectionSocketFactory(
+              getSslContext(),
+              getProtocols(),
+              getCipherSuites(),
+              SSLConnectionSocketFactory.BROWSER_COMPATIBLE_HOSTNAME_VERIFIER));
+    }
+
+    if (useBasicAuth()) {
+      httpClientBuilder.setDefaultCredentialsProvider(getCredentialsProvider());
+      httpClientBuilder.addInterceptorFirst(new PreemptiveAuth(new BasicScheme()));
+    }
```

Comment:
Lines 310-312 represent the core changes. If DDF is configured to use basic auth with Solr, the Solr client will always add basic auth headers. 

(I was hoping I could convince SolrJ to add the headers. However, to do it at the Solr layer, you have to set the basic auth properties on every instance of SolrRequest. That would have meant overriding every method in SolrClient that created a new instance of SolrRequest. It was only 4 or 5 methods but seemed really obtrusive, and likely to break if SolrJ changed at all. Digging down one layer deeper in the Apache HTTP Client layer was cleaner. )